nx_parallel/interface.py

```diff
@@ -35,18 +36,31 @@ class ParallelGraph:
 
     __networkx_backend__ = "parallel"
 
-    def __init__(self, graph_object):
-        self.graph_object = graph_object
+    def __init__(self, graph_object=nx.Graph()):
```

Comment:
We can't use `nx.Graph()` as the default value for a function because it will be the same object for all calls of this function. It is instantiated when the function is imported, not when it is run. (this is true for lists and dicts too.) So the typical pattern is:
```suggestion
    def __init__(self, graph_object=None):
        if graph_object is None:
            graph_object = nx.Graph()
```